core/store/src/db.rs

```diff
@@ -422,6 +429,19 @@ impl Database for RocksDB {
     }
 }
 
+fn next_prefix(prefix: &[u8]) -> Option<Vec<u8>> {
+    let mut prefix = prefix.to_vec();
+    for digit in prefix.iter_mut().rev() {
+        *digit = digit.wrapping_add(1);
+        if *digit != 0 {
+            return Some(prefix);
+        }
```

Comment:
if I understand correctly: 
next_prefix("a\xff\xff") would be "b\0\0", right ? 

shouldn't it be "b" ? 

can you add a unittest just for the next_prefix function ?